internal/sdkprovider/service/kafkatopic/kafka_topic_test.go

```diff
@@ -122,6 +122,64 @@ func TestAccAivenKafkaTopic_custom_timeouts(t *testing.T) {
 	})
 }
 
+func TestAccAivenKafkaTopic_with_owner_and_description(t *testing.T) {
+	resourceName := "aiven_kafka_topic.foo"
+	rName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.TestAccPreCheck(t) },
+		ProtoV6ProviderFactories: acc.TestProtoV6ProviderFactories,
+		CheckDestroy:             testAccCheckAivenKafkaTopicResourceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKafkaTopicWithOwnerAndDesctiptionResource(rName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "topic_description", fmt.Sprintf("test-acc-topic-desc-%s", rName)),
+					resource.TestCheckResourceAttrSet(resourceName, "owner_user_group_id"),
+				),
+			},
+		},
+	})
+}
+
+func testAccKafkaTopicWithOwnerAndDesctiptionResource(name string) string {
+	return fmt.Sprintf(`
+data "aiven_organization" "foo" {
+  name = "%s"
+}
+
+resource "aiven_organization_user_group" "foo" {
+  name            = "test-acc-u-grp-%s"
+  organization_id = data.aiven_organization.foo.id
+  description     = "test"
+}
+
+data "aiven_project" "foo" {
+  project = "%s"
+}
+
+resource "aiven_kafka" "bar" {
+  project                 = data.aiven_project.foo.project
+  cloud_name              = "google-europe-west1"
+  plan                    = "startup-2"
+  service_name            = "test-acc-sr-%s"
+  maintenance_window_dow  = "monday"
+  maintenance_window_time = "10:00:00"
+}
+
+resource "aiven_kafka_topic" "foo" {
+  project             = data.aiven_project.foo.project
+  service_name        = aiven_kafka.bar.service_name
+  topic_name          = "test-acc-topic-%s"
+  topic_description   = "test-acc-topic-desc-%s"
+  owner_user_group_id = aiven_organization_user_group.foo.group_id
+  partitions          = 3
+  replication         = 2
+}
+
+
+`, os.Getenv("AIVEN_ORGANIZATION_NAME"), name, os.Getenv("AIVEN_PROJECT_NAME"), name, name, name)
+}
```

Comment:
Hey. Can you please add a new topic to an existing test? I mean spinning out a whole new kafka for a single topic is kinda too much. What do you think?

Reply:
Ah, ok yeah makes sense. I understood wrong :)